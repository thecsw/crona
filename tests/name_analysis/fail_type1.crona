main : int (a : int, b : byte) {
    a = b;
}