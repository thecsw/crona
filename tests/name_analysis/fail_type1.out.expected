main(int,byte->int):int(a(int):int, b(byte):byte){
	a(int) = b(byte);
}